### PR TITLE
Update Flow plugin dependency

### PR DESCRIPTION
**Pull request by taooceros**
Replace old fuzzy search with the one provided by api

### CODE REVIEW

File: Flow.Launcher.Plugin.Everything.csproj [hunk 1]
```diff
@@ -118,14 +118,14 @@
...
     <PackageReference Include="Droplex" Version="1.0.4" />
-    <PackageReference Include="Flow.Launcher.Plugin" Version="1.3.1" />
-    <PackageReference Include="JetBrains.Annotations" Version="2019.1.3" />
+	<PackageReference Include="JetBrains.Annotations" Version="2019.1.3" />
     <PackageReference Include="System.Runtime" Version="4.3.1" />
   </ItemGroup>
 
   <ItemGroup>
-    <Reference Include="OLD_Flow.Launcher.Infrastructure">
+    <Reference Include="Flow.Launcher.Infrastructure">
```
Comment (jjw24) [inline Flow.Launcher.Plugin.Everything.csproj:128]: is this still needed?

File: Flow.Launcher.Plugin.Everything.csproj [hunk 2]
```diff
@@ -118,14 +118,14 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="Flow.Launcher.Plugin" Version="1.4.0" />
     <PackageReference Include="Droplex" Version="1.0.4" />
-    <PackageReference Include="Flow.Launcher.Plugin" Version="1.3.1" />
-    <PackageReference Include="JetBrains.Annotations" Version="2019.1.3" />
+	<PackageReference Include="JetBrains.Annotations" Version="2019.1.3" />
```
Comment (jjw24) [inline Flow.Launcher.Plugin.Everything.csproj:123]: is there an extra space in this?

File: Flow.Launcher.Plugin.Everything.csproj (hunk 1 above)
Comment (taooceros) [reply Flow.Launcher.Plugin.Everything.csproj:128]: There are still quite a lot of code that needs this.
1. The JSONStorage (which is not supposed to be in intrastracture)
2. ISavable interface
3. Log
4. maybe some more

File: Flow.Launcher.Plugin.Everything.csproj (hunk 2 above)
Comment (taooceros) [reply Flow.Launcher.Plugin.Everything.csproj:123]: a yes, I will remove it

Comment (taooceros) [reply Flow.Launcher.Plugin.Everything.csproj:123]: Oh there isn't one. No worry

File: Flow.Launcher.Plugin.Everything.csproj (hunk 1 above)
Comment (taooceros) [reply Flow.Launcher.Plugin.Everything.csproj:128]: I do agree that we shall gradually move this need to the api

**Comment by jjw24** [comment]
version bump

**Review by jjw24 — APPROVED** [review]

**Comment by jjw24** [comment]
@taooceros do you know if they can now use acronym search with Everything?

**Pull request merged**